src/Pages/Sidemenu.jsx

```diff
@@ -0,0 +1,40 @@
+import Box from "@mui/material/Box";
+import List from "@mui/material/List";
+import Divider from "@mui/material/Divider";
+import ListItem from "@mui/material/ListItem";
+import ListItemText from "@mui/material/ListItemText";
+import { useNavigate } from "react-router-dom";
+
+export default function Sidemenu() {
+
+  const navigate = useNavigate();
+
+  const handleClick = (e, text) => {
+    console.log({ text });
+    navigate(text);
+  };
+
+  return (
+    <div>
+      <Box
+        sx={{
+          width: 250,
+          // height: "100vh", position: "fixed", top: 0, left: 0, zIndex: 1
+        }}
+      >
+        <List>
+          {["dashboard", "portofolio", "register", "news"].map((text, index) => (
```

Comment:
This is a decent approach to rendering a list of navigators, but hardcoding these here is a bad idea for maintainability. I would move this array to your `constants.js` file, or some other similar file, and import it for use here. Then if it needs to change or be added to, you can go directly to the file rather than hunting for it in your components

Reply:
I might change the whole approach and just remove the MUI element and use React to create and style it myself
but I will make sure to hide the array if keeping I that way